app/services/encrypted_attribute.rb

```diff
@@ -7,7 +7,6 @@ def self.new_user_access_key(cost: nil, key: nil)
     key ||= env.attribute_encryption_key
     cost ||= env.attribute_cost
     new_key = load_or_build_user_access_key(cost: cost, key: key).dup
-    new_key.random_r = Pii::Cipher.random_key
```

Comment:
was this just unnecessary because `random_r` was already being set?

Reply:
Yeah, this was a weird thing where `random_r` was set when the key was initialized and wasn't reset when it was built. Now we are setting `random_r` when we build the user access key.